doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -702,6 +702,12 @@ Modem libraries
 
   * Rename the nRF91 socket offload layer from ``nrf91_sockets`` to ``nrf9x_sockets`` to reflect that the offload layer is not exclusive to the nRF91 Series SiPs.
 
+  * Updated:
+
+    * The :ref:`modem_trace_module` to let the trace thread sleep when the modem trace level is set to :c:enum:`NRF_MODEM_LIB_TRACE_LEVEL_OFF` using the :c:func:`nrf_modem_lib_trace_level_set` function, and the trace backend returns ``-ENOSPC``. The trace thread wakes up when another trace level is set.
+    * The RTT trace backend to return ``-ENOSPC`` when the RTT buffer is full. This allows the trace thread to sleep to save power.
+
+
```

Comment:
I think we need to add the snippet information (nrf91-modem-trace-rtt) on this as well : https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf91/nrf91_snippet.html .
Also, a changelog entry for the snippet?

Reply:
added.